tests/harvest/test_harvest_cli.py

```
@@ -4,31 +1,41 @@
+        ),
+    ],
+)
+def test_args(args_line, expected) -> None:
+    parser = get_arg_parser()
+    assert parser is not None
+    ns = parser.parse_args(args_line.split(" "))
+    args = vars(ns)
+    expected_set = set(expected.items())
+    args_set = set(
+        (k, tuple(v) if isinstance(v, list) else v) for k, v in args.items()
+    )
```

Ruff

Unnecessary generator (rewrite as a `set` comprehension)

Rewrite as a `set` comprehension

(C401)